portfolio/src/main/webapp/blogs.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+	<meta name="Feiyang (Kathy) Yu's Blog Posts" content="Hidden talent blog posts of Feiyang (Kathy) Yu">
+	<link rel="stylesheet" type="text/css" href="/style.css">
+    <script src="/script.js"></script>
+	<title>Kathy's Blog Posts</title>
+  </head>
+  
+  <body>
+    <!--- Main section contains the blog post content -->
+    <main>
+      <h1>Kathy's Blog Posts</h1>
+      <nav role="navigation" aria-label="navigation bar">
+        <a href="#blog1">Blog 1</a><br>
+        <a href="#blog2">Blog 2</a><br>
+      </nav>
+	  <p>Welcome to my blog posts on hidden talents!</p>         
+            
+	  <h3 id="blog1">Hidden Talent NO.1</h3>
+	  <p>I know how to play badminton. I've always enjoyed the sport since I was a child, and I used to play it every week when I was in middle school. I haven't played it for a while now, so I'm quite rusty. But badminton is still my favorite sport regardless.</p>
+
+	  <h3 id="blog2">Hidden Talent NO.2</h3>
+	  <p>I've never had a nose bleed.</p>
+    </main>
+
+	<nav role="navigation" aria-label="navigation bar">
+	  <a href="/index.html">Go back to homepage</a>
+      <a href="/gallery.html">Check out gallery</a>
+      <a href="/index.html#contact_me">Contact Me</a>
+	</nav>
+
+    <footer>Last modified on <time>2020-05-26</time>.</footer>
+
+    </body>
+</html>
```

Comment:
Add a new line after this

Reply:
To confirm, are you referring to the practice that there should be a newline at the end of all source code files? I should add a newline after the closing html tag. Thanks!